pkg/doc/signature/jsonld/testdata/json_vc_with_proper_contexts

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
these files should have file extensions. e.g., this is a .jsonld

Reply:
will use `.jsonld` for for json-ld files and `.rdf` for rdf files
plz correct if you have other ideas for rdf